Doc/library/signal.rst

```diff
@@ -503,3 +503,37 @@ be sent, and the handler raises an exception. ::
 
    signal.alarm(0)          # Disable the alarm
 
+Note on SIGPIPE
+---------------
+
+Do not set set :const:`SIGPIPE`'s disposition to :const:`SIGDFL`
+in order to avoid the :code:`BrokenPipeError: [Errno 32] Broken
+pipe` error message when you pipe your program into head(1).
+
+This will cause your program to exit in unexpected ways because Python
+will exit if a write to any socket which is the remote side has closed
+or dropped occurs.
```

Comment:
Would make more sense without _is_. Also may be easier to understand with _occurs_ closer to _write_: “a write occurs to any socket which the remote side has closed or dropped.”